docs/dev/developer-guides/transactions/transactions.md

```diff
@@ -0,0 +1,62 @@
+# Overview
```

Comment:
Not sure if we should repeat the same knowledge that could be read in the [Ethereum documentation](https://ethereum.org/en/developers/docs/transactions/). 

IMHO we should only provide the difference between Ethereum and us. Especially, since most of our features is a subset of the features that Ethereum supports, the users should be generally advised to understand how Ethereum works. 

If you think that we should anyway repeat the same information as on Ethereum documentation page, then we should do it the zkSync way, e.g. do not tell users that only EOAs can sign transactions. The account abstraction feature that will be supported enables contract accounts to initiate transactions on their own.

Reply:
Asides from doing it the zkSync way(which is only concerned with transactions in this case), perhaps I would link to relevant sources that are relatable.